src/Microsoft.DotNet.Build.Tasks/PackageFiles/notsupported.targets

```diff
@@ -41,6 +41,7 @@
       <GenAPIArgs>$(GenAPIArgs) -libPath:"@(_referencePathDirectories)"</GenAPIArgs>
       <GenAPIArgs>$(GenAPIArgs) -out:"$(NotSupportedSourceFile)"</GenAPIArgs>
       <GenAPIArgs>$(GenAPIArgs) -throw</GenAPIArgs>
+      <GenAPIArgs Condition="'$(GeneratePlatformNotSupportedAssembly_Message)' != ''">$(GenAPIArgs) -exceptionMessage:"$(GeneratePlatformNotSupportedAssembly_Message)"</GenAPIArgs>
```

Comment:
I think you need to update this given your change here: https://github.com/dotnet/buildtools/pull/1451/files#diff-e82cfcbeb709508bae5e486ffa8dcbb6R268